dedibox-console/account/how-to/resolve-abuse.mdx

```diff
@@ -1,32 +1,64 @@
+  validation: 2024-02-19
   posted: 2020-03-12
 categories:
   - account
 ---
 
-Abuse tickets are complaints regarding the hosting of copyrighted content or engaging in illegal activities using Scaleway services.
+Abuse tickets are complaints regarding the use of Scaleway Dedibox services for illegal behavior or hosting of illegal content. Abuses for the same issue on the same resource within a given timeframe are grouped by case.
+Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the lock of their services.
+However, Scaleway reserves the right to immediately suspend the infringing services if it is deemed necessary and proportionate to keep Scaleway’s network safe or to abide by legal requirements.
+
+If you encounter any issue with the resolution of your abuses, please contact the abuse team by opening a [support ticket](https://console.scaleway.com/support/tickets/create) directly from your console.
 
 <Message type="requirement">
-  - You have an account and are logged into the [Dedibox console](https://console.online.net) 
+  - You have an account and are logged into the [Dedibox console](https://console.online.net)
 </Message>
 
-1. Click your username, next to _logged in as_, in the top right corner of the Dedibox console. Then, select **Abuses** from the drop-down menu. A list of your abuse tickets displays.
-2. Click **Details** to view the specifics of the complaint.
-3. Take note of the details provided and proceed with the necessary steps to address and resolve the issue. If you need guidance on resolving outgoing DDoS attacks from your server, refer to our documentation [how to prevent outgoing DDoS](/compute/instances/reference-content/preventing-outgoing-ddos/) for assistance.
-4. Describe the measures you have taken and provide an answer to the author of the ticket once the issue is solved. Click **Resolve** to mark the ticket as closed.
-    <Message type="note">
-      Refer to [our abuse notice](https://www.scaleway.com/en/abuse-notice/) for further information about Scaleway's abuse policy.
+1. Click your username, located next to “logged in as” in the top right corner of the Dedibox console. Then, select Abuses from the drop-down menu. A list of your abuse tickets and cases displays.
+2. Click Details to view the specifics of the complaint.
+3. Take note of the details provided and proceed with the necessary steps to address and resolve the issue.
+    <Message type="tip">
+      If you need guidance on resolving outgoing DDoS attacks from your server, refer to our documentation [how to prevent outgoing DDoS](https://www.scaleway.com/en/docs/compute/instances/reference-content/preventing-outgoing-ddos/) for assistance.
     </Message>
+    * In case you receive an abuse for hosting illegal content (including copyright), please delete the content from Scaleway’s infrastructure promptly.
```

Comment:
```suggestion
    * In case you receive an abuse ticket for hosting illegal content (including copyright), please delete the content from Scaleway’s infrastructure promptly.
```